### PR TITLE
HW ver 1.0 ?..

### PR DESCRIPTION
**Pull request by Purpletta**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+from typing import Optional
+import streamlit as st
+import torch
+from diffusers import (
+    StableDiffusionPipeline,
+)
```
Comment (SpirinEgor) [inline src/main.py]: Use 1 line here 🙃

File: src/main.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            st.image(output_image)
+        else:
+            st.markdown("No output created yet.")
+    st.write("NOTE: if you see a completely black image after generation, try increasing the \"Number inference steps\" "
+             "in the settings")
+
+
+if __name__ == "__main__":
+
+
```
Comment (SpirinEgor) [inline src/main.py]: Why so many blank lines?

**Review by SpirinEgor — COMMENTED** [review]
Great app, all looks quite good.
Let's prettify it a little bit, I suggest checking your code with linters, e.g. pep8 or black.

File: src/main.py (hunk 1 above)
Comment (Purpletta) [reply src/main.py]: Fixed!

File: src/main.py (hunk 2 above)
Comment (Purpletta) [reply src/main.py]: Idk( corrected it

**Review by SpirinEgor — APPROVED** [review]

**Comment by Purpletta** [comment]
Should I merge pr?

**Comment by SpirinEgor** [comment]
I approved your PR, you can do whatever your want with the repo 🙃

**Pull request merged**